net-nds/389-ds-base/files/389-ds.initd-r1

```diff
@@ -0,0 +1,90 @@
+#!/sbin/openrc-run
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+DIRSRV_EXEC="/usr/sbin/ns-slapd"
+PID_DIRECTORY="/var/run/dirsrv"
+LOCK_DIRECTORY="/var/lock/dirsrv"
+DIRSRV_CONF_DIR="/etc/dirsrv"
+DS_INSTANCES=${DIRSRV_CONF_DIR}/slapd-*
+F389DS_INSTANCES=""
+
+depend() {
+	need net logger
```

Comment:
ebuild doesn't have `logger` dependency, this init script failed to start on my system without it.